src/amo/components/HeadMetaTags/index.js

```diff
@@ -6,50 +6,90 @@ import { connect } from 'react-redux';
 import { compose } from 'redux';
 
 import { getCanonicalURL } from 'amo/utils';
+import { CLIENT_APP_ANDROID } from 'core/constants';
+import translate from 'core/i18n/translate';
 import type { AppState } from 'amo/store';
+import type { I18nType } from 'core/types/i18n';
 
 type Props = {|
+  appendDefaultTitle?: boolean,
   date?: Date | null,
-  description: string,
+  description?: string | null,
```

Comment:
I want the prop to be optional because some pages (the categories page actually) don't have a description at all.

I want the prop to allow `null` because it is possible that *some* category pages don't have a description (`string | null`). I don't want to do complex prop computation in the parent component.

That is why it needs both.